src/pages/Detail/Detail.js

```diff
@@ -1,7 +1,201 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import ModalShare from './Modal/ModalShare';
+
+const options = {
+  center: new window.kakao.maps.LatLng(37.5063, 127.05367),
+  level: 2,
+};
 
 const Detail = () => {
-  return <div></div>;
+  const KakaoMapContainer = useRef(null);
+
+  useEffect(() => {
+    const markerPosition = new window.kakao.maps.LatLng(37.5063, 127.05367);
+    const marker = new window.kakao.maps.Marker({
+      position: markerPosition,
+    });
```

Comment:
```suggestion
    const marker = new window.kakao.maps.Marker({
      position: new window.kakao.maps.LatLng(37.5063, 127.05367);
    });
```
굳이 변수 선언해서 담을필요 없어보입니다!